specification/dimensions/chargefrequency.md

```diff
@@ -0,0 +1,40 @@
+# Charge Frequency
+
+Charge Frequency indicates how often a charge would occur. 
```

Comment:
Would it make sense to add a reference to ChargePeriodStart/End here? Maybe something like this...?

> _While Charge Frequency indicates whether a charge will reoccur, Charge Period Start and Charge Period End specify the period for which this charge applies. This can typically be used to also understand the recurrence period (e.g., monthly, yearly)._